### ISSUE TITLE
Do not call `LoadPackage( "gauss", ... )' in the package file

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/Thelma#13]
The following issue is reported in test logs:
```
#I  Thelma: Do not call `LoadPackage( "gauss", ... )' in the package file
#I          /circa/scratch/gap-jenkins/workspace/GAP-pkg-update-stable-quicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/packagesload/label/kovacs/GAP-pkg-update-stable-snapshot/pkg/Thelma-1.02/lib/internal_functions.gi,
#I          use `IsPackageMarkedForLoading' instead
```

**Comment by vlaver** [comment]
> The following issue is reported in test logs:
> 
> ```
> #I  Thelma: Do not call `LoadPackage( "gauss", ... )' in the package file
> #I          /circa/scratch/gap-jenkins/workspace/GAP-pkg-update-stable-quicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/packagesload/label/kovacs/GAP-pkg-update-stable-snapshot/pkg/Thelma-1.02/lib/internal_functions.gi,
> #I          use `IsPackageMarkedForLoading' instead
> ```

Done, made the corresponding changes.

**Comment by alex-konovalov** [comment]
Thanks. I looked at b6a4c3bf1f232a00a5fd843df1eaea190295f402 and actually you don't need that line at all since Gauss is a required package.

Also `IsPackageMarkedForLoading` is not used that way. It returns a boolean and does not perform any action.

**Comment by vlaver** [comment]
"you don't need that line at all" - deleted it, sorry.

**Comment by alex-konovalov** [comment]
Closed by d8698fd5112385181c3c3e1665225913db821f48 - thanks
